fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +491,38 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    c_seqnum = self.get_current_seq_number(lsa)
+                    if not c_seqnum:
+                        # If LSA wasn't in the lsdb: process it normally
+                        log.debug('LSA was not in the LSDB yet')
+
+                    else:
+                        log.debug('Seqnum for LSA in lsdb: %s)', c_seqnum)
+                        log.debug('Seqnum for newly arrived LSA: %s)', new_seqnum)
+                        if new_seqnum == c_seqnum:
+                            log.debug("DUPLICATE seqnum for new LSA: %s", lsa)
+
+                            if hdr.age >= MAX_LS_AGE:
+                                log.debug("Wrapping seqnum around for LSA: %s", lsa)
+
+                            else:
+                                # What about in this case?
+                                pass
```

Comment:
Not sure I fully understand your issue there. Could paste a complete log somewhere ? Does the graph eventually gets build once the network has converged ?